src/components/blog-post-details/blog-post-details.js

```diff
@@ -0,0 +1,92 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { withBlogService } from '../hoc';
+import { bindActionCreators } from 'redux';
+import { fetchPostDetails } from '../actions';
+import Loader from '../loader';
+import ErrorIndicator from '../error-indicator';
+
+import './blog-post-details.css';
+
+class BlogPostDetails extends Component {
+  state = {
+    inputValue: ''
+  };
+
+  onCommentAdd = e => {
+    e.preventDefault();
+    const { id, fetchPostDetails } = this.props;
+
+    const postId = parseInt(id);
+
+    this.props.blogService
```

Comment:
`this.props.blogService.addComment(postId, this.state.value)` this logic must be placed as side-effect with `redux-thunk` or `redux-saga`, etc.

If you want to chain several calls of actions - just return your promise from it.